armi/bookkeeping/db/database3.py

```diff
@@ -52,6 +52,7 @@
     Tuple,
     Type,
 )
+import gc
```

Comment:
Can you move this up to between lines 35 and 36, so the std lib imports are still in alphabetical order?

Thanks!